### PR TITLE
Metabase local and ALCS CLI 

### PR DESCRIPTION
**Pull request by pdittaro**
Configuration for local metabase development, plus ALCS CLI POC to supplement development efforts.

### CODE REVIEW

**Comment by pdittaro** [comment]
alcs-cli

File: bin/alcs [hunk 1]
```diff
@@ -0,0 +1,660 @@
...
+#   printf "Hello, World!\\n"
+# }
+#
+###############################################################################
+
+# Docker Section #############################################################
+
+# --------------------------------------------------------------------- docker
+
+describe "docker" <<HEREDOC
```
Comment (pdittaro) [inline bin/alcs:602]: Here's the docker related functionality, including metabase

File: bin/alcs [hunk 2]
```diff
@@ -0,0 +1,660 @@
...
+# Example:
+#   _debug printf "Debug info. Variable: %s\\n" "$0"
+__DEBUG_COUNTER=0
+_debug() {
+  if ((${_USE_DEBUG:-0}))
+  then
+    __DEBUG_COUNTER=$((__DEBUG_COUNTER+1))
+    {
+      # Prefix debug message with "bug (U+1F41B)"
+      printf "🐛  %s " "${__DEBUG_COUNTER}"
```
Comment (mhuseinov) [inline bin/alcs:73]: I like this bug

File: bin/README.md [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+if [ -d "$HOME/Workspace/alcs/bin" ] ; then
+  PATH="$HOME/Workspace/alcs/bin:$PATH"
+fi
+
+```
+
+
+## Commands
+
+* `alcs docker [up/down] (--metabase)`
```
Comment (mhuseinov) [inline bin/README.md:18]: Is up/down optional parameters?

Comment (pdittaro) [reply bin/README.md:18]: No, they're required. If you do `alcs docker` without it will print the usage().

Comment (pdittaro) [reply bin/README.md:18]: I'll put more into the Readme if we decide to use a CLI tool like this.

**Review by mhuseinov — APPROVED** [review]
🔥

**Pull request merged**